tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml

```diff
@@ -18,3 +18,16 @@ tests:
       - nrf9160dk_nrf9160_ns
       - qemu_cortex_m3
     extra_args: CONF_FILE=prj_tinycrypt.conf
+  fast_pair.crypto.mbedtls.core:
+    platform_allow:
+      nrf52840dk_nrf52840
+    integration_platforms:
+      - nrf52840dk_nrf52840
+    extra_args: CONFIG_BT_FAST_PAIR_TEST_CRYPTO_EXT=n
+  fast_pair.crypto.tinycrypt.core:
+    platform_allow:
+      nrf52840dk_nrf52840 qemu_cortex_m3
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - qemu_cortex_m3
+    extra_args: CONF_FILE=prj_tinycrypt.conf CONFIG_BT_FAST_PAIR_TEST_CRYPTO_EXT=n
```

Comment:
No need for this test definition if you agree with my previous review comments

Reply:
Discussed with @alstrzebonski. We can simplify the configuration of both fp_crypto library and the unit test and rely on optimizing out unused functions (I noticed that memory consumption is not increased, so we should be fine).